tests/unit-tests/test-class-quiz.php

```diff
@@ -697,6 +697,37 @@ public function testGetUserQuestionAnswer() {
 		// make sure that after a reset this function returns false
 	}//end testGetUserQuestionAnswer()
 
+	/**
+	 * This tests Woothemes_Sensei()->quiz->get_user_question_answer without user_id argument.
+	 */
+	public function testGetUserQuestionAnswerWithoutUserId() {
+		// setup the data needed for the assertions
+		$test_user_id           = wp_create_user( 'studentGetQuestionAnswer', 'studentGetQuestionAnswer', 'studentGetQuestionAnswer@test.com' );
+		$test_lesson_id         = $this->factory->get_random_lesson_id();
+		$test_quiz_id           = Sensei()->lesson->lesson_quizzes( $test_lesson_id );
+		$test_user_quiz_answers = $this->factory->generate_user_quiz_answers( $test_quiz_id );
+		$files                  = $this->factory->generate_test_files( $test_user_quiz_answers );
+		Sensei()->quiz->save_user_answers( $test_user_quiz_answers, $files, $test_lesson_id, $test_user_id );
+		$random_question_id  = array_rand( $test_user_quiz_answers );
+		$users_saved_answers = Sensei()->quiz->get_user_answers( $test_lesson_id, $test_user_id );
+
+		$this->assertNull(
+			Sensei()->quiz->get_user_question_answer( $test_lesson_id, $random_question_id ),
+			'Should return null when has no user_id argument and is not logged in'
+		);
+
+		// Logged user
+		wp_set_current_user( $test_user_id );
+		$question_answer = Sensei()->quiz->get_user_question_answer( $test_lesson_id, $random_question_id );
+
+		$this->assertEquals(
+			$users_saved_answers[ $random_question_id ],
+			$question_answer,
+			'Should return the correct answer if has no user_id argument but user is logged in'
+		);
+
+	}//end testGetUserQuestionAnswerWithoutUserId()
```

Comment:
Can we remove the //end comments?

Reply:
I took the opportunity to update all comment standards in this file: https://github.com/Automattic/sensei/pull/3016/commits/a3c6d6d41a6f34633c21ebfdb0618dfb809c56f0